src/transformers/pipelines/zero_shot_classification.py

```diff
@@ -221,6 +222,13 @@ def _forward(self, inputs):
         candidate_label = inputs["candidate_label"]
         sequence = inputs["sequence"]
         model_inputs = {k: inputs[k] for k in self.tokenizer.model_input_names}
+        # `XXXForSequenceClassification` models should not use `use_cache=True` even if it's supported
```

Comment:
Can we use `self.framework` here instead of checking for attributes? Something like
```py
model_forward = self.model.forward if self.framework == "pt" ekse self.model.call
if "use_cache" in set(inspect.signature(model_forward).parameters().keys():
    model_inputs["use_cache"] = False
```

Reply:
sure!